README.md

```diff
@@ -101,6 +101,17 @@ Dor.find("druid:pv820dk6668").destroy
 %w[pv820dk6668 rn653dy9317 xb482bw3979 hj185vb7593 hv992ry2431].each{ |pid| Dor.find("druid:#{pid}").destroy }
 ```
 
+### DOR virtual-merge
+
```

Comment:
Can you put a 1 sentence explanation of virtual-merge here?

Reply:
I don't really know.  I'm just porting from https://github.com/sul-dlss/dor-utils/

Reply:
"A tool to merge multiple objects into a parent object via virtual object structure, namely the externalFile element in contentMetadata." (which is clear as mud, but better than nothing)

Reply:
I'm writing stuff in ticket now.  See last comment:  https://github.com/sul-dlss/dor-utils/issues/17

Are we sure this is the ONLY use of dor-utils?
Are we sure putting this functionality in argo is the way forward?  dor-utils is on lyberservices-prod box and may need access to stuff there ...

I'm not opposed to moving it, but ... should we move it NOW?